boot/build.go

```diff
@@ -324,3 +328,14 @@ func friendlySize(size float64) string {
 
 	return fmt.Sprintf("%0.1f %s", size, unit)
 }
+
+func FindSpringCloudBindingsJars(jars []libjvm.MavenJAR) *libjvm.MavenJAR {
```

Comment:
You could consider making this func generic, e.g. FindExistingDependency and pass in the name you want to match.

If you don't think the version is needed you could simplify this func and simply return a bool

Reply:
yes, good idea